### ISSUE TITLE
Helm dask notes seem to have a typo in setting the environment variable

### ISSUE THREAD

**Issue by twiecki** [dask/dask#4253]
I followed the guide at: http://docs.dask.org/en/latest/setup/kubernetes-helm.html

Running `helm install stable/dask` instructs me to run:
![image](https://user-images.githubusercontent.com/674200/49107365-cf762a00-f285-11e8-9924-d7b003af308b.png)

to set the right env variables. Note that the variable is called `DASK_SCHEDULER` while dask seems to look for `DASK_SCHEDULER_ADDRESS`, so 
![image](https://user-images.githubusercontent.com/674200/49107414-f3d20680-f285-11e8-944f-234b80b7b652.png)
was not working for me unless I fixed that.

**Comment by twiecki** [comment]
Actually that doesn't seem to be right either as it requires `tcp://` in front it seems. Is that env variable supposed to be set automatically? Because it was not.

**Comment by mrocklin** [comment]
Yup, that looks like a genuine issue.  We used to use the `DASK_SCHEDULER`
environment variable, but changed this to `DASK_SCHEDULER_ADDRESS` at some
point in the last year.  Changing this upstream in the helm chart would be
good.

I plan to update that chart sometime in the next couple weeks (after a
release soonish) and so can probably handle it then.  If anyone wants to
beat me to it that would also be welcome :)

**Comment by mrocklin** [comment]
Thanks for reporting @twiecki

**Comment by mrocklin** [comment]
The tcp:// is desired, but will be prepended by default if not present.

**Comment by twiecki** [comment]
Actually I'm not quite sure I'm doing it right:
`You can create a notebook and create a Dask client from there. The DASK_SCHEDULER_ADDRESS environment variable has been populated with the address of the Dask scheduler. This is available in Python in the config dictionary.`

I ran ipython locally instead, it works fine when I connect to the jupyter lab server and run it there. Or is it supposed to locally as well?

**Comment by mrocklin** [comment]
You need to provide the Client object with a network address that connects to the `dask-scheduler` pod.  Within Kubernetes the `bald-eel-scheduler:8786` address should work.  Outside of Kubernetes you'll need to find an address that is externally visible.  This should be accessible with the `helm` or `kubectl` CLI, though I admit that the exact command escapes me at the moment.

**Comment by twiecki** [comment]
if I run `export DASK_SCHEDULER_ADDRESS=tcp://$DASK_SCHEDULER:8786` it works fine with just instantiating `Client`, although then running something fails locally then with:

```
In [3]: >>> def square(x):
   ...:         return x ** 2
   ...:
   ...: >>> def neg(x):
   ...:         return -x
   ...:
   ...: >>> A = client.map(square, range(10))
   ...: >>> B = client.map(neg, A)
   ...: >>> total = client.submit(sum, B)
   ...: >>> total.result()
   ...:
   ...:
---------------------------------------------------------------------------
CancelledError                            Traceback (most recent call last)
<ipython-input-3-0b952c058b1d> in <module>()
      8 B = client.map(neg, A)
      9 total = client.submit(sum, B)
---> 10 total.result()

~/anaconda3/lib/python3.6/site-packages/distributed/client.py in result(self, timeout)
    195             six.reraise(*result)
    196         elif self.status == 'cancelled':
--> 197             raise result
    198         else:
    199             return result

CancelledError: sum-5c00f435c56b61583526e7a19aa3a2bd
```

**Comment by mrocklin** [comment]
You should check worker logs.  But my guess is that you have mismatched versions between your local client library and the version used in the helm chart.  You might consider running `client.get_versions(check=True)`.  If I'm right then you'll get some exception.

**Comment by twiecki** [comment]
`TypeError: versions() got an unexpected keyword argument 'packages'`, I guess that means yes? :)

**Comment by mrocklin** [comment]
Yes.   If you're within a decent range of versions then you'll get a nicely printed out version of which packages are out of sync.  If you're mismatched enough then you get that :/

The docker image in the stable helm chart is pretty old.  You might consider bumping it up with a small config file.  Something like the following:

```yaml
scheduler:
  image: "daskdev/dask"
  tag: 0.19.4

worker:
  image: "daskdev/dask"
  tag: 0.19.4

notebook:
  image: "daskdev/dask-notebook"
  tag: 0.19.4
```

Then `helm upgrade bald-eagle -f my-config.yaml`

It looks like there isn't a docker image newer than that up on docker hub.

**Comment by mrocklin** [comment]
I've tagged a 0.20.2 on docker hub.  Should be built and up in a while.

**Comment by martindurant** [comment]
> If you're mismatched enough then you get that :/

Presumably check_versions should be resilient to this (for the future) and give a sensible error if it failed to fetch information successfully.

**Comment by mrocklin** [comment]
Yes, that would be ideal.  Anyone want to submit a PR fixing that?

**Comment by twiecki** [comment]
If I run `helm repo update` and `helm install stable/dask` should I get dask containers with 0.20.2? Because the version still fails.
